tests/shared/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -505,6 +505,36 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
     assert(templStat("class C extends (() => Int)").syntax == "class C extends (() => Int)")
   }
 
+  test("package object e extends D") {
```

Comment:
could you please tests with `given` and `new` anonymous? and whatever else can potentially use a template.

Reply:
Added tests in a separate suite for Scala 3 (I had some issues with the implicit dialect, so decided that it's safer)